document some more Release Manager assumptions (#344)
apache/openwhisk-release: docs/release_instructions.md

# Instructions for Release Managers

## Release Manager Initial Setup

If you have never served as a Release Manager (or it has been a while
since you have done it), first read the
[Release Manager Setup](release_manager_setup.md) instructions.
We have scripts that automate many of the tasks a Release Manager must
perform, but these scripts assume you have followed the instructions
in the Release Manager Setup before you start executing them.

## Non-PMC Release Managers

Actually publishing a release is an official act of the OpenWhisk

> Project Management Committee (PMC).  If you an OpenWhisk Committer,

Reviewer: ```suggestion
Project Management Committee (PMC).  If you are an OpenWhisk Committer,
```

but not an OpenWhisk PMC member, you will need to pair-up with a PMC member to
accomplish some of the steps.  In brief, any step that adds or removes
an artifact from a public distributions channel must be executed by

> an OpenWhisk  PMC member.

Reviewer: ```suggestion
an OpenWhisk PMC member.
```

./local_verify.sh ../release-configs/<MY_RELEASE_CONFIG>.json
```

TODO:  We should also run Apache Rat (https://creadur.apache.org/rat/)

> as part of local_verify.sh; for now it is a recommended best practice

Reviewer: ```suggestion
as part of `local_verify.sh`; for now it is a recommended best practice
```

### Dockerhub updates

> **Only a PMC member can push images to the openwhisk organization on dockerhub**

Reviewer: ```suggestion
**Only a PMC member can push images to the OpenWhisk organization on Docker Hub**
```

Reviewer: heh I see `dockerhub` is throughout... ignore this nit :)
